target/linux/ath79/nand/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -25,6 +25,17 @@ case "$FIRMWARE" in
 		;;
 	esac
 	;;
+"ath10k/pre-cal-pci-0000:00:00.0.bin")
+	case $board in
+	glinet,gl-x1200-nor|\
+	glinet,gl-x1200-nor-nand)
+		caldata_extract "art" 0x5000 0x2f20
+		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
+			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
+		rm /lib/firmware/ath10k/QCA9888/hw2.0/board-2.bin
```

Comment:
Why is this coping and removing needed?

Reply:
I have tested that ath10k cannot correctly identify the mac address in caldata <0x5006 0x6> without removing board-2.bin.
So I refer to comfast,cf-e313ac device, and remove board-2.bin.

Reply:
Thanks for the explanation. Is the mac address correctly set when you define the reference to the calibration data and mac address in the device tree like you did it now?

Reply:
Yes, I have tested it and now the mac address is correct.